interfaces/builtin/serial_port.go

```diff
@@ -75,7 +75,8 @@ func (iface *serialPortInterface) String() string {
 //  - ttySCX (NXP SC16IS7xx serial devices)
 //  - ttyMSMX (Qualcomm msm7x serial devices)
 //  - ttyHSX (Qualcomm GENI based QTI serial cores)
-var serialDeviceNodePattern = regexp.MustCompile("^/dev/tty(mxc|USB|ACM|AMA|XRUSB|S|O|SC|MSM|HS)[0-9]+$")
+//  - ttyGS (USB gadget serial devices)
```

Comment:
nitpick: for consistency with the above, please write `ttyGSX`

Reply:
thanks & updated